### ISSUE TITLE
Error in livewire init for v0.2.6

### ISSUE THREAD

**Issue by 2kodes** [calebporzio/livewire#176]
**Describe the bug**
After creating a new project and adding the directive @livewireAssets(['base_url'=>"http://127.0.0.1:8000"]) livewire gives an error in the js console as follows

```
 livewire.js?id=7d4e531b4b2407078242:1 Uncaught TypeError: Cannot set property 'onMessage' of undefined
    at new e (livewire.js?id=7d4e531b4b2407078242:1)
    at new e (livewire.js?id=7d4e531b4b2407078242:1)
    at (index):91
```

**To Reproduce**
1.  create a new laravel project `php  composer create-project --prefer-dist laravel/laravel test_livewire`
2. install livewire to the project  `php composer require calebporzio/livewire`
3. add the livewireAssets directive to the /resources/views/welcome.blade.php file     
`@livewireAssets(['base_url'=>"http://127.0.0.1:8000"])`
4. type `php artisan serve` to serve the project with artisan
5. Open the console tab in the browser. Access the browser url `http://127.0.0.1:8000` and reload or refresh the page to see the error

**Expected behavior**
I expected livewire to load and be initialized successfully with no error messages

**Screenshots**
![image](https://user-images.githubusercontent.com/2084398/63263776-060f2d00-c279-11e9-937b-c527456fb9d6.png)

**Desktop (please complete the following information):**
 - Browser [ chrome Version 75.0.3770.100 (Official Build) (64-bit) ]

**Additional context**
Giving and using the base url as from a webserver folder also yielded the same error. I havent tried with a subdomain though

**Comment by intellow** [comment]
You don't need to put a 'base_url' parameter. Just use `@livewireAssets`

**Comment by 2kodes** [comment]
> You don't need to put a 'base_url' parameter. Just use `@livewireAssets`

@intellow that was to emphasize that it doesnt work when serving the app with `artisan serve`

i added extra info in the additional context for my setup.  i am not serving with artisa but from a web server folder

**Comment by calebporzio** [comment]
Good call @2kodes  - this should be fixed in 0.2.7
